common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExecutorDiskUtils.java

```diff
@@ -41,6 +48,13 @@ public static File getFile(String[] localDirs, int subDirsPerLocalDir, String fi
         localDir, String.format("%02x", subDirId), filename));
   }
 
+  /** Returns whether the OS is Windows. */
+  @VisibleForTesting
+  static boolean isWindows() {
+    String os = System.getProperty("os.name");
+    return os.startsWith("Windows");
+  }
```

Comment:
Use existed `Utils.isWindows`?

Reply:
The pre-existing method is in `core` module, `network-shuffle` doesn't depend on it.

Reply:
I see. Then, could we also use `org.apache.commons.lang3.SystemUtils.IS_OS_WINDOWS` for simplicity?

Reply:
Sound good.